.github/workflows/dependencies.yml

```diff
@@ -6,7 +6,6 @@ on:
       - '**/Cargo.toml'
       - 'deny.toml'
   push:
-    branches: [main, staging, trying]
```

Comment:
Does this mean it will run our daily cron jobs on every branch?

Reply:
no, that was a limit only on the push trigger, the schedule didn't change